content/en/news/releases/1.6.x/announcing-1.6/change-notes/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Change Notes
+description: Istio 1.6 release notes.
+weight: 10
+---
+
+## User facing changes
+- ***Added*** experimental support for the [Kubernetes Service APIs](https://github.com/kubernetes-sigs/service-apis).
+- ***Added*** the new [Workload Entry](https://preliminary.istio.io/docs/reference/config/networking/workload-entry/) resource. This will allow easier configuration for non-Kubernetes workloads to join the mesh.
+- ***Added*** support for configuring Gateway topology settings like number of trusted proxies deployed in front
+- ***Added*** experimental mesh-wide tracing configuration API. This API provides control of trace sampling rates, the maximum tag lengths for URL tags, and custom tags extraction for all traces within the mesh. 
+- ***Added*** experimental request classification filter support. This feature enables operators to configure new attributes for use in telemetry, based on request information. A primary use case for this feature is labeling of traffic by API method.  TODO: doc
+- ***Added*** Wasm extension distribution support. TODO: docs
```

Comment:
@mandarjog on behalf of Doug

Reply:
Lets nuke this for now. I dont remember seeing any proper framework or PRs to allow users to configure wasm extensions for their gateways/sidecars.

Reply:
@rshriram it might not meet your bar for "proper", but the work has definitely been on-going: https://github.com/envoyproxy/envoy-wasm/commit/356b66b357c49b52b28335431d73508dfea3442c. we should mark this experimental, but I do believe the intention is to have this tried out in some capacity in this release.

Reply:
@douglas-reid there s a "TODO: docs" - is there a related document?

Reply:
I don't have one. This is why I added @mandarjog in my original suggestion :). Maybe @bianpengyuan knows of one in development?

Reply:
`distribution support` seems to be too much. We should make it `remote load support`. I am planning to add a section for that in this page: https://deploy-preview-7242--preliminary-istio.netlify.app/docs/reference/config/telemetry/telemetry_v2_with_wasm/.

Reply:
@jacob-delgado Please remove this line. We run into some issue with remote load feature.

Reply:
Removed